src/test/java/DeadlineTest.java

```diff
@@ -0,0 +1,54 @@
+import duke.*;
```

Comment:
Perhaps you can consider listing explicitly all the classes of the Duke Package required for this testing?

Reply:
yes resolved thank u